packages/react-native-gesture-handler/android/build.gradle

```
@@ -158,9 +158,15 @@ android {
                     arguments "-DREACT_NATIVE_DIR=${REACT_NATIVE_DIR}",
                             "-DREACT_NATIVE_MINOR_VERSION=${REACT_NATIVE_MINOR_VERSION}",
                             "-DANDROID_STL=c++_shared",
-                            "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
+                            "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON",
+                            // turn off build IDs for reproducibility
```

```suggestion
                            // Turn off build IDs for reproducibility.
```

Also it would be great if you could add some more information (not just one keyword - _reproducibility_) 😅

---

I've added some more info, though I wasn't sure if you'd prefer a link to the guide explicitly suggesting to turn off build IDs or the RB site with a more general overview

---

> I wasn't sure if you'd prefer a link to the guide explicitly suggesting to turn off build IDs or the RB site with a more general overview

First one sounds better. It is easier to reach home page from specific subpage than the other way around 😅